app/styles/common/fontChina.sass

```diff
@@ -2,3 +2,5 @@
 @import url(//fonts.loli.net/css?family=Arvo:400,700)
 @import url(//fonts.loli.net/css?family=Open+Sans:400,300,700&subset=latin,latin-ext,cyrillic,vietnamese,cyrillic-ext,greek-ext,greek)
 @import url(//fonts.loli.net/css?family=Open+Sans+Condensed:700&subset=latin,latin-ext,cyrillic-ext,greek-ext,greek,vietnamese,cyrillic)
+
+@import url('//fonts.googleapis.com/css?family=Arvo|Roboto+Mono|Work+Sans&display=swap&subset=cyrillic,cyrillic-ext,greek,greek-ext,latin-ext,vietnamese')
```

Comment:
We can't load googleapi fonts in china. This will need to be replaced with the proxy mirror that the other fonts are using.

Reply:
Good catch, updated and verified URL works